utf8parse/src/lib.rs

```diff
@@ -91,3 +91,43 @@ impl Parser {
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::io::prelude::*;
```

Comment:
Can this import be changed to import only what is needed? I have an allergy to glob imports :wink: